modules/ROOT/pages/migration-process.adoc

```diff
@@ -8,22 +8,22 @@ endif::[]
 //TODO: LINK TO MULE 4 SECTIONS FOR ALL THESE STEPS.
 After learning how to prepare for the migration, it is important to understand the high-level migration steps:
 
-. Add all required modules, such as the Database connector, to the Anypoint Studio 7 palette.
+. Add all required modules, such as the Database connector, to the Anypoint Studio (Studio) 7 palette.
 +
 Notes:
 +
 * You can add modules through the Studio 7 palette. Other modules are available in Anypoint Exchange.
 +
-* All Mule projects are Mavenized by default when you add the modules to your project.
+* All Mule projects are mavenized by default when you add the modules to your project.
```

Comment:
Should Mavenized be capped?

Reply:
I looked it up in the maven doc, it's not capitalized by them...